contracts/Voter.sol

```diff
@@ -239,7 +312,15 @@ contract Voter is IVoter {
         isAlive[_gauge] = true;
         _updateFor(_gauge);
         pools.push(_pool);
-        emit GaugeCreated(_gauge, msg.sender, _internal_bribe, _external_bribe, _pool);
+        Pair(_pool).setHasGauge(true); // may need to switch to IPair?
```

Comment:
@duncan4123 yeah you should use IPair if possible

Reply:
@t0rbik